### DISCUSSION TITLE
Bisakah satu file di Route dapat dipanggil dengan nama yang berbeda-beda?

### DISCUSSION THREAD

**Discussion by bungfrangki** [mecha-cms/mecha#288] [Q&A]
Misalnya saya punya file JS bernama **environment.js**, yang disimpan pada web filehost.com
dan saya ingin menjalankan JS tersebut pada beberapa website, tetapi dengan nama yang berbeda-beda.
Misalnya, pada web web1.com saya ingin panggil file tersebut dengan filehost.com/assets/_web1.com_/environment.js,
lalu pada web2.com saya ingin panggil dengan filehost.com/assets/_web2.com_/environment.js, di web3.com ingin saya jalankan dengan filehost.com/assets/_web3.com_/environment.js
dan seterusnya.

**Comment by taufik-nurrohman** [comment]
File harus eksis di tempat lain kemudian dimuat menggunakan ekstensi [route](https://github.com/orgs/mecha-cms/discussions/266) dengan _cache header_ tertentu.
